src/pages/components/card-section.mdx

```diff
@@ -49,4 +55,50 @@ Note: This uses the underlying component "Card group". You can find more informa
 
 To see the list of all character count standards, go to this <a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank" rel="noreferrer noopener nofollow">link</a>.
 
+## Card section with images
+Card section with images brings images into each card in the section. Like Card section, this components is typically used for presenting resources or links, but also presents images in each card.
+
+![Card section images](../../images/pattern/card-section/card-section-images.jpg)
+<Caption>An example of Card section with images</Caption>
+
+### Resources for Card section with images
```

Comment:
Let's change this to the Resource link with the multi prop included, so that we're automated. 🤖